ui-common/src/components/modals/ReplayRequestModal/ReplayRequestModal.tsx

```diff
@@ -239,7 +239,7 @@ const ReplayRequestModal: React.FC<ReplayRequestModalProps> = ({ isOpen, onClose
                                 <span className={styles.sectionHeader}>RESPONSE</span>
                             </AccordionSummary>
                             <AccordionDetails>
-                                <AutoRepresentation representation={response} color={entryData.protocol.backgroundColor} />
+                                <AutoRepresentation representation={response} color={entryData.protocol.backgroundColor} defaultTab={TabsEnum.Response} />
```

Comment:
why do you need to pass the defaultTab from outside if it doesnt dynamically change?
wouldnt it be better to declare the response tab as default inside the component?

Reply:
if params name was openedTab it was making more sense?

i didn't want to change the current behavior and only setting the behavior for the current component

Reply:
Oh I see, it's the same component for request and response.

so nvm, you can leave it like that